jme3-core/src/main/java/com/jme3/renderer/RenderManager.java

```diff
@@ -104,6 +105,7 @@ public class RenderManager {
     private TechniqueDef.LightMode preferredLightMode = TechniqueDef.LightMode.MultiPass;
     private int singlePassLightBatchSize = 1;
     private MatParamOverride boundDrawBufferId=new MatParamOverride(VarType.Int,"BoundDrawBuffer",0);
+    private Function<Geometry, Boolean> renderFilter;

```

Comment:
Since the filter always returns a boolean value, you can use a `Predicate` instead of a `Function`. 

<html>
<body>
<!--StartFragment-->

Sr. No. | Key | Function | Predicate
-- | -- | -- | --
1 | Basic | It can take 2 type parameters First one represents input type argument type and second one represents return type. | It can take one type parameter which represents input type or argument type.
2 | Return Type | It can return any type of value. | It can only return boolean value
3 | Method | It has abstract method apply(). | It has abstract method test().
4 | Use Case | It can be used to implement conditional checks | It can be used for the transformation and to the return values.

<!--EndFragment-->
</body>
</html>

Reply:
Done :+1: